docs/CSBR.md

```diff
@@ -432,18 +1277,59 @@ copyright: |
 
 # 8. COMPLIANCE AUDIT AND OTHER ASSESSMENTS
 
+The CA and/or all Signing Services MUST, at all times:
+
+1.  Comply with all laws applicable to its business and the Certificates it issues in each jurisdiction where it operates,
+2.  Comply with these Requirements,
+3.  Comply with the audit requirements set forth in this section, and
+4.  If a CA, be licensed as a CA in each jurisdiction where it operates, if licensing is required by the law of such jurisdiction for the issuance of Certificates.
+

```

Comment:
I believe this is a nice fit for the existing 14.2.2, until we further discuss and possible remove this language.

`In all cases, the CA MUST contractually obligate each Delegated Third Party to comply with all applicable requirements in these Requirements and to perform them as required of the CA itself. The CA MUST enforce these obligations and internally audit each Delegated Third Party’s compliance with these Requirements on an annual basis. `